src/Dhall/Core.hs

```diff
@@ -352,6 +357,14 @@ data Expr s a
     | TextLit (Chunks s a)
     -- | > TextAppend x y                           ~  x ++ y
     | TextAppend (Expr s a) (Expr s a)
+    -- | >  FilePath                                ~  FilePath
+    | FilePath
```

Comment:
Well, I don’t know, are you sure? I feel that it will be somewhat more ambiguous. Also, currently it nicely corresponds to the Haskell type, as other primitive types (e.g. `Integer`, `Text`).